packages/angular/src/generators/application/lib/add-e2e.ts

```diff
@@ -2,7 +2,7 @@ import type { Tree } from '@nrwl/devkit';
 import { joinPathFragments } from '@nrwl/devkit';
 import type { NormalizedSchema } from './normalized-schema';
 
-import { cypressProjectGenerator } from '@nrwl/cypress';
+import { cypressProjectGenerator } from '@nrwl/cypress/src/generators/cypress-project/cypress-project';
```

Comment:
Do we need this change? Is it because something imported/exported in the package entry file causes issues?

Reply:
i change the import to root cypress.